python/examples/calendar/schema.py

```diff
@@ -0,0 +1,89 @@
+from typing import Literal, NotRequired, TypedDict, Annotated
+def Doc(s: str) -> str: return s
```

Comment:
This is okay for now - in the future, we can just import https://typing-extensions.readthedocs.io/en/latest/#Doc